packages/app-store/stripepayment/components/EventTypeAppCardInterface.tsx

```diff
@@ -37,7 +37,7 @@ const EventTypeAppCard: EventTypeAppCardComponent = function EventTypeAppCard({
       .trim();
   return (
     <AppCard
-      returnTo={WEBAPP_URL + asPath}
+      returnTo={WEBAPP_URL + pathname}
```

Comment:
There are many places where we have started using pathname instead of asPath. asPath has query params also in it. So, we should ideally replace this with pathname plus searchParams/

But I have scanned most of them and they might be fine as we are just losing query param